src/client/sandbox/node/window.ts

```diff
@@ -76,6 +76,9 @@ const SANDBOX_DOM_TOKEN_LIST           = 'hammerhead|sandbox-dom-token-list';
 const SANDBOX_DOM_TOKEN_LIST_OWNER     = 'hammerhead|sandbox-dom-token-list-owner';
 const SANDBOX_DOM_TOKEN_LIST_UPDATE_FN = 'hammerhead|sandbox-dom-token-list-update';
 
+const IS_PROXY_OBJECT        = 'hammerhead|is-proxy-object';
```

Comment:
const IS_PROXY_OBJECT_INTERNAL_PROP_NAME  = 'hammerhead|is-proxy-object|internal-prop-name';
const IS_PROXY_OBJECT_INTERNAL_PROP_VALUE  = 'hammerhead|is-proxy-object|internal-prop-value';